### PR TITLE
Add on_water property on practice

### PR DESCRIPTION
**Pull request by gutard**

### CODE REVIEW

File: geotrek/trekking/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -240,8 +243,18 @@ msgstr "Zone de sensibilité"
...
 msgstr "POIs"
 
-msgid "services"
-msgstr "services"
+msgid "Services"
+msgstr "Services"
+
+#, fuzzy
+#| msgid "Stages"
+msgid "Signages"
```
Comment (camillemonchicourt) [inline geotrek/trekking/locale/fr/LC_MESSAGES/django.po]: Mistake between stages and signages ?

File: geotrek/trekking/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -240,8 +243,18 @@ msgstr "Zone de sensibilité"
...
+msgstr "Étapes"
+
+#, fuzzy
+#| msgid "Disabled Infrastructure"
+msgid "Infrastructures"
+msgstr "Aménagement handicapés"
```
Comment (camillemonchicourt) [inline geotrek/trekking/locale/fr/LC_MESSAGES/django.po]: Shouldn't it be "Disabled infrastructures" as initially, instead of "Infrastructures" ?

**Comment by gutard** [comment]
Pas de panique, c'était juste une démo lors d'une formation ;)

**Comment by camillemonchicourt** [comment]
OK !!! :smile: :+1:

**Pull request closed without merging**